docs/cita/hacking/chain-gov.md

```diff
@@ -3,4 +3,7 @@ id: chain-gov
 title: 链上治理
 ---
 
-TODO
+更多信息请查看[链上治理 GitHub Repo]
+
+
+[链上治理 GitHub Repo]: https://github.com/cryptape/appchain-gov-general
```

Comment:
`appchain` -> `cita`?

Reply:
> `appchain` -> `cita`?

@rainchen Should we modify the repo title and domain name？Can you fix it?

Reply:
ok, let me fix it

Reply:
here is the  new repo URL: https://github.com/cryptape/cita-gov-general-demo